src/host/process_launcher.h

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/messaging.h"
+#include "enclave/interface.h"
+
+#include <chrono>
+#include <queue>
+#include <unordered_map>
+#include <uv.h>
+
+namespace asynchost
+{
+  class ProcessLauncher
+  {
+    static constexpr size_t max_processes = 8;
```

Comment:
That could really benefit from being a launch-time option.

Reply:
Sure, but since it's an experimental feature let's wait for feedback first.

Reply:
I think the best way to know that it's an experimental feature would be for it to be spelled out as such in the changelog ;)